test-clients/src/main/java/com/hedera/services/bdd/suites/fees/AllBaseOpFeesSuite.java

```diff
@@ -0,0 +1,251 @@
+ * limitations under the License.
+ * ‍
+ */
+
+import com.google.protobuf.ByteString;
+import com.hedera.services.bdd.spec.HapiApiSpec;
+import com.hedera.services.bdd.suites.HapiApiSuite;
+import com.hederahashgraph.api.proto.java.TokenSupplyType;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.time.Instant;
+import java.util.List;
+
+import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.burnToken;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoTransfer;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.mintToken;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.tokenAssociate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.tokenCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.tokenFreeze;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.tokenUnfreeze;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.wipeTokenAccount;
+import static com.hedera.services.bdd.spec.transactions.token.TokenMovement.movingUnique;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.newKeyNamed;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.validateChargedUsdWithin;
+import static com.hedera.services.bdd.suites.utils.MiscEETUtils.metadata;
+import static com.hederahashgraph.api.proto.java.TokenType.NON_FUNGIBLE_UNIQUE;
+
+public class AllBaseOpFeesSuite extends HapiApiSuite {
+	private static final Logger log = LogManager.getLogger(AllBaseOpFeesSuite.class);
+
+	private static final double ALLOWED_DIFFERENCE_PERCENTAGE = 0.01;
+
+	private static final String TREASURE_KEY = "treasureKey";
+	private static final String FUNGIBLE_COMMON_TOKEN = "fungibleCommonToken";
+
+	private static final String ADMIN_KEY = "adminKey";
+	private static final String SUPPLY_KEY = "supplyKey";
+	private static final String FREEZE_KEY = "freezeKey";
+	private static final String WIPE_KEY = "wipeKey";
+	private static final String KYC_KEY = "kycKey";
+
+	private static final String CIVILIAN_ACCT = "civilian";
+
+	private static final String UNIQUE_TOKEN = "nftType";
+
+	private static final String BASE_TXN = "baseTxn";
+
+
+	private static final double EXPECTED_UNFREEZE_PRICE_USD = 0.001;
+	private static final double EXPECTED_FREEZE_PRICE_USD = 0.001;
+	private static final double EXPECTED_NFT_MINT_PRICE_USD = 0.001;
+	private static final double EXPECTED_NFT_BURN_PRICE_USD = 0.001;
+	private static final double EXPECTED_NFT_WIPE_PRICE_USD = 0.001;
+
+
+	public static void main(String... args) {
+		new AllBaseOpFeesSuite().runSuiteSync();
+	}
+
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return allOf(List.of(new HapiApiSpec[] {
+				baseNftFreezeUnfreezeChargedAsExpected(),
```

Comment:
Great idea! 🎉 

**But**...if this suite ran in CI, it wouldn't pass. 😝 

This is because the fee schedules in file `0.0.111` of the saved state we use in CI won't match your updated JSON.

However, if you update [_test-clients/src/main/resource/FeeSchedule.json_](https://github.com/hashgraph/hedera-services/blob/master/test-clients/src/main/resource/FeeSchedule.json) with your changes, CI will be happy, since it [begins](https://github.com/hashgraph/hedera-services/blob/master/test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java#L96) with an `uploadDefaultFeeSchedules()` operation.

Reply:
Yeah, I will remember to make sure FeeSchedule.json to have the same values as that in feeSchedule.json, as CI run won't have services node to download the fee schedules from.